tools/roslaunch/src/roslaunch/depends.py

```diff
@@ -142,6 +143,14 @@ def _parse_launch(tags, launch_file, file_deps, verbose, context):
                 (name, val) = v
                 context['arg'][name] = val
         elif tag.tagName == 'include':
+            if tag.attributes.has_key('if'):
+                val = resolve_args(tag.attributes['if'].value, context)
+                if not convert_value(val, 'bool'):
+                    continue
+            elif tag.attributes.has_key('unless'):
+                val = resolve_args(tag.attributes['unless'].value, context)
+                if convert_value(val, 'bool'):
+                    continue
```

Comment:
`rospack` 2.4.1 has propagated to shadow-fixed already: http://repositories.ros.org/status_page/ros_lunar_default.html?q=rospack

Other jobs have turned green.